homeassistant/components/homekit/type_covers.py

```diff
@@ -91,47 +92,50 @@ def update_state(self, new_state):
             self._flag_state = False
 
 
-@TYPES.register("WindowCovering")
-class WindowCovering(HomeAccessory):
-    """Generate a Window accessory for a cover entity.
+class WindowCoveringBase(HomeAccessory):
+    """Generate a base Window accessory for a cover entity.
 
-    The cover entity must support: set_cover_position.
+    This class is used for WindowCoveringBasic and
+    WindowCovering
     """
 
-    def __init__(self, *args):
-        """Initialize a WindowCovering accessory object."""
+    def __init__(self, *args, category):
+        """Initialize a WindowCoveringBase accessory object."""
         super().__init__(*args, category=CATEGORY_WINDOW_COVERING)
 
-        self._homekit_target = None
+        self.features = self.hass.states.get(self.entity_id).attributes.get(
+            ATTR_SUPPORTED_FEATURES, 0
+        )
+        self.supports_stop = self.features & SUPPORT_STOP
```

Comment:
`supports_tilt` is prefixed with `_` but this one is not.

Reply:
fixed